docs/contributing/commit-signing.mdx

```diff
@@ -101,6 +101,17 @@ Refer to
   your terminal. Refer to
   [this troubleshooting document](https://gist.github.com/paolocarrasco/18ca8fe6e63490ae1be23e84a7039374)
   for more help with this error
+- If you have already committed your changes and forgot to configure gpg signing you can fix it by
+  doing a rebase like the following targeting the hash of the previous commit to the one you did (it
+  will sign all commits from there):
+
+  ```
```

Comment:
Please add bash to enable syntax highlighting.

Reply:
Nice catch thanks!